src/travis-config.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * @license
+ * Copyright (c) 2018 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import * as fse from 'fs-extra';
+import {safeDump, safeLoad} from 'js-yaml';
+import * as path from 'path';
+
+const TRAVIS_CONFIG = '.travis.yml';
```

Comment:
nit: we generally don't follow the CAPS_LOCK_SNAKE_CASE for constants, `travisConfig` is fine